hazelcast/src/test/java/com/hazelcast/internal/serialization/impl/compact/schema/CompactSchemaReplicationStressTest.java

```diff
@@ -153,7 +157,11 @@ public void run() {
         }
 
         private void replicateSchemas() {
-            for (int i = 0; i < SCHEMA_COUNT ; i++) {
+            for (int i = 1; i <= SCHEMA_COUNT; i++) {
+                if (i % 10_000 == 0) {
```

Comment:
is 10000 too high? Total is 42000 and hence only 4 logs. May be you change to 2000 to see more  granular progress.

Reply:
There are 30 drivers, so I didn't want to fill the logs with frequent updates.

I believe this is enough to see the progress